### PR TITLE
Add dynomite

### PR DESCRIPTION
**Pull request by Veetaha**
cc @softprops

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -914,6 +914,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * CouchDB [[couchdb](https://crates.io/keywords/couchdb)]
     * [chill-rs/chill](https://github.com/chill-rs/chill) [[couchdb](https://crates.io/crates/chill)] — A Rust client for the CouchDB REST API [<img src="https://api.travis-ci.org/chill-rs/chill.svg?branch=master">](https://travis-ci.org/chill-rs/chill)
     * [Sofa](https://github.com/66Origin/sofa) — an interface to CouchDB HTTP REST API for stable rust
+  * [DynamoDB](https://aws.amazon.com/dynamodb) [[dynamodb](https://crates.io/keywords/dynamodb)]
+    * [softprops/dynomite](https://github.com/softprops/dynomite) - A library for strongly-typed and convenient interaction with `rusoto_dynamodb` [<img src="https://github.com/softprops/dynomite/workflows/Main/badge.svg"/>](https://github.com/softprops/dynomite/actions)
```
Comment (palfrey) [inline README.md:918]: Please add the branch information to the build badge. See https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#example-using-the-branch-parameter

File: README.md [hunk 2]
```diff
@@ -914,6 +914,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * CouchDB [[couchdb](https://crates.io/keywords/couchdb)]
     * [chill-rs/chill](https://github.com/chill-rs/chill) [[couchdb](https://crates.io/crates/chill)] — A Rust client for the CouchDB REST API [<img src="https://api.travis-ci.org/chill-rs/chill.svg?branch=master">](https://travis-ci.org/chill-rs/chill)
     * [Sofa](https://github.com/66Origin/sofa) — an interface to CouchDB HTTP REST API for stable rust
+  * [DynamoDB](https://aws.amazon.com/dynamodb) [[dynamodb](https://crates.io/keywords/dynamodb)]
```
Comment (palfrey) [inline README.md:917]: Also this should be `https://aws.amazon.com/dynamodb/` (note the ending slash) as the other link 301's to that

Comment (Veetaha) [reply README.md]: Wow, thanks :D

File: README.md (hunk 1 above)
Comment (Veetaha) [reply README.md]: Yup, done

**Comment by Veetaha** [comment]
Sorry I had to rebase due to a merge conflict, whilst doing this I squashed the commit with new changes according to the review

**Comment by palfrey** [comment]
> Sorry I had to rebase due to a merge conflict, whilst doing this I squashed the commit with new changes according to the review

No worries. Thanks for the changes.

**Pull request merged**